test/Microsoft.Azure.WebJobs.Host.UnitTests/Configuration/WebJobsExtensionOptionsConfigurationTests.cs

```diff
@@ -19,7 +21,7 @@ public void Configuration_IsBound()
                 { "azurewebjobs:extensions:testext:testvalue2", "test2" },
             };
 
-            TestOptions options = ConfigureOptions<TestOptions>(config, "testext");
+            TestOptions options = ConfigureOptions<TestOptions, TestExtensionConfigProvider>(config);
```

Comment:
I don't see any new tests for the new functionality? E.g. I'd expect a test verifying that when ConfigureOptions is called for an extension, the new ExtensionOptionsInfo gets registered

Reply:
Added Functions Testing instead.